addon/utils/_private/rdfa-parser/rdfa-parser.ts

```diff
@@ -1297,7 +1304,8 @@ export class RdfaParser<N> {
   ) {
     // Validate IRIs
     if (
-      (subject.termType === 'NamedNode' && subject.value.indexOf(':') < 0) ||
+      // More hacking... Does including '#' make sense?
+      (subject.termType === 'NamedNode' && subject.value.indexOf(':') < 0 && subject.value.indexOf('#')) ||
```

Comment:
I don't think this is necessarily correct, as IRIs do not need to contain a hash symbol, that is just an often used delimiter.

Reply:
This isn't requiring them to contain `#`, the logic is the other way around. If it does not contain `:` and does not contain `#`, then it's not an IRI, so ignore it. This is here to catch the case that we have in article structure nodes where the subject is `#` (or at least, that's what we have after my hack to handle the nodes with `resource="#"`).

Reply:
Ok I see what you mean, was a bit confused. Would if not be necessary to negate the `indexOf('#')` then?

Reply:
Oh, good spot. It was meant to mimic the `:` test, so `subject.value.indexOf('#') < 0`. What I wrote is nonsense as it's only false if it's testing a string that starts with `#`...